locals.tf

```diff
@@ -36,36 +30,45 @@ nginx:
               values:
               - "gvisor"
     tolerations:
-      - effect: NoSchedule
-        key: sandbox.gke.io/runtime
-        operator: Equal
-        value: gvisor
+    - effect: NoSchedule
+      key: sandbox.gke.io/runtime
+      operator: Equal
+      value: gvisor
+%{endif}
+
+nginx:
+  loadBalancerIP: ${var.load_balancer_ip == "" ? "~" : var.load_balancer_ip}
+  privateLoadBalancer: ${var.cluster_type == "private" ? true : false}
+  perserveSourceIP: true
 
 astronomer:
+%{if var.enable_gvisor == "true"}
   houston:
     config:
```

Comment:
@ianstanton and I chatted about this earlier, but we need to nest `deployments` in here between `config` and `helm`, per https://github.com/astronomer/houston-api/blob/master/config/default.yaml#L277-L427

Reply:
@sjmiller609 I'll be adding and testing this today